src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -23,67 +22,7 @@
     <PackageReference Include="runtime.$(ToolsRID).Microsoft.DotNet.ILCompiler" Version="$(MicrosoftDotNetILCompilerVersion)" />
   </ItemGroup>
 
-  <!-- Needed for the amd64 -> amd64 musl cross-build to pass the target flag. -->
-  <Target Name="_FixIlcTargetTriple"
```

Comment:
This should be handled by https://github.com/dotnet/runtime/blob/b1f63dbfa4216f67b8f8d5aabdb33834d70989f6/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets#L46 (since it's required by public consumption as well)